pkg/constants/http_test.go

```diff
@@ -78,7 +78,21 @@ func TestURLConstruction(t *testing.T) {
 			execute: func() string {
 				return URLTxProof(1, 2)
 			},
-			expectedURL: "/ledger/proof/1?idx=2",
+			expectedURL: "/ledger/proof/tx/1?idx=2",
+		},
+		{
+			name: "URLDataProof deleted false",
+			execute: func() string {
+				return URLDataProof(1, "db1", "key", false)
+			},
+			expectedURL: "/ledger/proof/data/db1/key?block=1",
```

Comment:
I am okay with the usage of `block` rather than `block_number`. To maintain consistency, we need to use `block` for other HTTP requests too. I think any of us can address this in a separate PR.

Reply:
Agreed. I will open issue and lets discuss it.

https://github.com/IBM-Blockchain/bcdb-server/issues/111